week5/feet_meters.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+
+'''
+Write a module that contains the following two functions:
+#Convertsfromfeettometers
+deffootToMeter(foot):
+#Convertsfrommeterstofeet
+defmeterToFoot(meter):
+The formulas for the conversion are:
+foot = meter / 0.305
+meter = 0.305 * foot
+Write a test program that invokes these functions to display
+the following tables:
+Feet Meters|MetersFeet
+1.0 0.305 |20.0 66.574
+2.0 0.610 |26.0 81.967
+...
+9.0 2.745 |60.0 196.721
+10.0 3.050 |66.0 213.115
+
+>>> footToMeter(1)
+0.305
+
+>>> meterToFoot(0.305)
+1.0
+'''
+
+
+def footToMeter(foot):
+    return foot * 0.305
+
+def meterToFoot(meter):
+    return meter / 0.305
+
+
+if __name__ == "__main__":
+
+    import doctest
+    doctest.testmod(raise_on_error=True)
+
+    print("Feet Meters | Meters Feet")
```

Comment:
If you want to try to test the `print` statement, you can read this [chapter](http://chimera.labs.oreilly.com/books/1230000000393/ch14.html#_testing_output_sent_to_stdout) from the excellent Python Cookbook, this is the same answer I put in the "Answer to students" part in the course.
But I warn you, there are lot of concepts that we have not seen yet.